modules/bibdocfile/lib/bibdocfile_config.py

```diff
@@ -69,3 +69,12 @@
 # CFG_BIBDOCFILE_DEFAULT_ICON_SUBFORMAT -- this is the default subformat used
 # when creating new icons.
 CFG_BIBDOCFILE_DEFAULT_ICON_SUBFORMAT = "icon"
+
+# CFG_BIBDOCFILE_SUBFORMATS_TRANSLATIONS -- a list (not dictionary, because
+# we want to preserve the order) that connects the different format sizes
+# (like 'small', 'medium', etc.) with internal format sizes (like 'icon-180', 'icon-640', etc.)
+CFG_BIBDOCFILE_SUBFORMATS_TRANSLATIONS = [
```

Comment:
Sure, I'm actually applying some fixes to that commit now (to not export the hidden files), so I will change the name as suggested.